pcsx2/Counters.cpp

```diff
@@ -270,6 +270,30 @@ static const char* ReportVideoMode()
 	}
 }
 
+Fixed100 GetVerticalFrequency()
+{
+	switch (gsVideoMode)
+	{
+	case GS_VideoMode::Uninitialized: // SYSCALL instruction hasn't executed yet, give some temporary values.
+		return 60;
+	case GS_VideoMode::PAL:
+		return EmuConfig.GS.FrameratePAL;
+	case GS_VideoMode::NTSC:
+		return EmuConfig.GS.FramerateNTSC;
+	case GS_VideoMode::HDTV_480P:
+		return 59.94;
+	case GS_VideoMode::HDTV_1080P:
+	case GS_VideoMode::HDTV_1080I:
```

Comment:
> I thought 1080i was 30fps?

Vertical frequency is the refresh rate controlled by the vertical blanking signal of PCRTC circuit, not related to fps. The actual frames pushed are 30fps as you had said earlier. (half of vertical frequency)

> PAL will be, surely?

What do you mean ? :P